packages/contracts-core/contracts/Replica.sol

```diff
@@ -359,6 +357,25 @@ contract Replica is Version0, NomadBase {
 
     // ============ Internal Functions ============
 
+    /**
+     * @notice Set optimistic timeout period for new roots
+     * @dev Called by owner (Governance) or at initialization
+     * @param _optimisticSeconds New optimistic timeout period
+     */
+    function _setOptimisticTimeout(uint256 _optimisticSeconds) internal {
+        // This allows us to initialize the value to be very low in test envs,
+        // but does not allow governance action to lower a production env below
+        // the safe value
+        uint256 _current = optimisticSeconds;
+        if (_current != 0 && _current > 1500) require(_optimisticSeconds >= 1500, "optimistic timeout too low");
```

Comment:
@anna-carroll please review this line carefully

Reply:
hmm prevents us from updating `optimisticSeconds` in test envs. should be acceptable tho.. let me think a sec

Reply:
it shouldn't prevent us from updating in test envs. It should be a ratchet. I.e. once above 1500, never below 1509

Reply:
what if 

if current is zero, allow anything 
if current is less than 1500, require it's greater than current
if current is geq 1500, require it's geq 1500

Reply:
yep, okay. good